tuf/store/offlinestore.go

```diff
@@ -38,7 +41,12 @@ func (es OfflineStore) RemoveMeta(name string) error {
 }
 
 // GetKey returns ErrOffline
-func (es OfflineStore) GetKey(role string) ([]byte, error) {
+func (es OfflineStore) GetKey(role string) (data.PublicKey, error) {
+	return nil, err
```

Comment:
Looking at this again, I shouldn't have use the name `err`. It's not breaking anything now but it's a package scope var. Do you mind updating it to `errOffline` or something else less generic?